java.lang.NoClassDefFoundError for AddInsideDataTagIntention

```
java.lang.NoClassDefFoundError: com/github/shiraji/databindinglayout/intentions/AddInsideDataTagIntention
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:760)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:642)
	at com.intellij.util.lang.UrlClassLoader._defineClass(UrlClassLoader.java:261)
	at com.intellij.util.lang.UrlClassLoader.defineClass(UrlClassLoader.java:257)
	at com.intellij.util.lang.UrlClassLoader._findClass(UrlClassLoader.java:226)
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClassInsideSelf(PluginClassLoader.java:119)
	at com.intellij.ide.plugins.cl.PluginClassLoader.tryLoadingClass(PluginClassLoader.java:73)
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:62)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:348)
	at com.intellij.openapi.extensions.AbstractExtensionPointBean.findClass(AbstractExtensionPointBean.java:42)
	at com.intellij.openapi.extensions.AbstractExtensionPointBean.instantiate(AbstractExtensionPointBean.java:63)
	at com.intellij.openapi.extensions.CustomLoadingExtensionPointBean.instantiateExtension(CustomLoadingExtensionPointBean.java:47)
	at com.intellij.codeInsight.intention.IntentionActionBean.instantiate(IntentionActionBean.java:76)
	at com.intellij.codeInsight.intention.impl.config.IntentionActionWrapper.getDelegate(IntentionActionWrapper.java:81)
	at com.intellij.codeInsight.intention.impl.config.IntentionActionWrapper.getFamilyName(IntentionActionWrapper.java:51)
	at com.intellij.codeInsight.intention.impl.config.IntentionActionWrapper.getFullFamilyName(IntentionActionWrapper.java:73)
	at com.intellij.codeInsight.intention.impl.config.IntentionManagerSettings.getFamilyName(IntentionManagerSettings.java:129)
	at com.intellij.codeInsight.intention.impl.config.IntentionManagerSettings.isEnabled(IntentionManagerSettings.java:142)
	at com.intellij.codeInsight.intention.impl.config.IntentionManagerImpl.getAvailableIntentionActions(IntentionManagerImpl.java:260)
	at com.intellij.codeInsight.daemon.impl.ShowIntentionsPass.getActionsToShow(ShowIntentionsPass.java:302)
	at com.intellij.codeInsight.daemon.impl.ShowIntentionsPass.getIntentionActionsToShow(ShowIntentionsPass.java:225)
	at com.intellij.codeInsight.daemon.impl.ShowIntentionsPass.doCollectInformation(ShowIntentionsPass.java:201)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:70)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass$1$1.run(PassExecutorService.java:444)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1181)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass$1.run(PassExecutorService.java:435)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:446)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:392)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:54)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.doRun(PassExecutorService.java:432)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:408)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:206)
	at jsr166e.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at jsr166e.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:858)
	at jsr166e.ForkJoinPool.scan(ForkJoinPool.java:1687)
	at jsr166e.ForkJoinPool.runWorker(ForkJoinPool.java:1642)
	at jsr166e.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:108)
Caused by: java.lang.ClassNotFoundException: com.github.shiraji.databindinglayout.intentions.AddInsideDataTagIntention PluginClassLoader[com.github.shiraji.databindinglayout, 1.0.9-SNAPSHOT]
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:64)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 40 more
```

This seems weird since `AddInsideDataTagIntention` is abstract and didn't register plugin.xml

--- a/src/main/kotlin/com/github/shiraji/databindinglayout/intentions/AddImportTagIntention.kt
+++ b/src/main/kotlin/com/github/shiraji/databindinglayout/intentions/AddImportTagIntention.kt
@@ -4,6 +4,7 @@ import com.intellij.openapi.editor.Editor
 import com.intellij.psi.xml.XmlTag
 
 class AddImportTagIntention : AddInsideDataTagIntention() {
+    override fun getText() = "Add <import> tag"
     override val tagName = "import"
     override val tagTemplate = "<import type=\"\"/>"
     override fun moveCaret(addedTag: XmlTag?, editor: Editor?) {

--- a/src/main/kotlin/com/github/shiraji/databindinglayout/intentions/AddInsideDataTagIntention.kt
+++ b/src/main/kotlin/com/github/shiraji/databindinglayout/intentions/AddInsideDataTagIntention.kt
@@ -13,7 +13,7 @@ import com.intellij.psi.xml.XmlFile
 import com.intellij.psi.xml.XmlTag
 
 abstract class AddInsideDataTagIntention : IntentionAction {
-    override fun getText() = "Add <$tagName> tag"
+    abstract override fun getText(): String
     override fun getFamilyName() = text
     override fun startInWriteAction() = true
     override fun isAvailable(project: Project, editor: Editor?, file: PsiFile?) = file.hasDatabindingLayout()

--- a/src/main/kotlin/com/github/shiraji/databindinglayout/intentions/AddVariableTagIntention.kt
+++ b/src/main/kotlin/com/github/shiraji/databindinglayout/intentions/AddVariableTagIntention.kt
@@ -4,6 +4,7 @@ import com.intellij.openapi.editor.Editor
 import com.intellij.psi.xml.XmlTag
 
 class AddVariableTagIntention : AddInsideDataTagIntention() {
+    override fun getText() = "Add <variable> tag"
     override val tagName = "variable"
     override val tagTemplate = "<variable name=\"\" type=\"\"/>"
     override fun moveCaret(addedTag: XmlTag?, editor: Editor?) {

